nms/app/packages/magmalte/grafana/dashboards/Dashboards.js

```diff
@@ -306,6 +330,70 @@ export const GatewayDBData: GrafanaDBData = {
   ],
 };
 
+export const SubscriberDBData: GrafanaDBData = {
+  title: 'Subscribers',
+  description:
+    'Metrics relevant to subscribers. Do not edit: edits will be overwritten. Save this dashboard under another name to copy and edit.',
+  templates: [networkTemplate, imsiTemplate, apnTemplate, msisdnTemplate],
+  rows: [
+    {
+      title: '',
+      panels: [
+        {
+          title: 'UE Data Usage In',
+          targets: [
+            {
+              expr:
+                'sum(ue_reported_usage{IMSI=~"$imsi", direction="down"}) by (IMSI, apn, msisdn)',
```

Comment:
The way these templates work is that the parameters like `$imsi` are filled in from the variable (in this case the one provided by `imsiTemplate`). So in this panel the only variable that will have any effect is `imsi`. Since it's like that for all the other panels, we should just get rid of any that aren't imsi.

That being said, the partner we're working with wanted to select on MSISDN instead of IMSI, so I think it would be better to use that one here to be consistent.